docs/requirements.md

```diff
@@ -18,8 +18,8 @@ The cluster running Codacy must satisfy the following requirements:
 
 -   The infrastructure hosting the cluster must be provisioned with the hardware and networking requirements described below
 -   The orchestration platform managing the cluster must be one of:
-    -   [Kubernetes](https://kubernetes.io/) **version 1.14.\*** to **1.20.\*** (1.16 recommended)
-    -   [MicroK8s](https://microk8s.io/) **version 1.14.\*** to **1.18.\*** (1.16 recommended)
+    -   [Kubernetes](https://kubernetes.io/) **version 1.14.\*** to **1.20.\*** (1.20 recommended)
+    -   [MicroK8s](https://microk8s.io/) **version 1.14.\*** to **1.18.\*** (1.20 recommended)
```

Comment:
We're recommending a version that's outside of the supported MicroK8s version. :thinking: 

See this pull request for more context: https://github.com/codacy/chart/pull/665

Was this solved in the meanwhile?

Reply:
We are testing 1.20 with microk8s also:
https://github.com/codacy/chart/blob/master/.circleci/config.yml#L687

In the latest version (3.3.0) already had this:
https://app.circleci.com/pipelines/github/codacy/chart/4738/workflows/e0ea77c8-6b12-42d6-8a72-570efd53efe8